docs/setup/asb/systemd/pull-and-build-asb.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Pull and build ASB from source
```

Comment:
I think that is too specific to our setup.

Reply:
That is what the service does at the moment. It's true that a service provider would not need that. 
It is not only this file that is concerned by this task, but also the timer, path-watcher and the restart service. Should I put those files into a separate folder?
It's still nice to have them in GitHub if only for us to track what is changing.